### DISCUSSION TITLE
Smart Meter Descriptor example EMH / eBZD

### DISCUSSION THREAD

**Discussion by Marc0803** [arendst/Tasmota#15008] [Ideas]
Hi,

does somebody have a smart meter descriptor example for my EMH eBZD? I would like to read the values with my ESP8266 and tasmota. Or some hints how to create a script for that meter?

Thanks a lot!

Marc

![screenshot_821](https://user-images.githubusercontent.com/75698604/156043833-ccce4c2e-9621-4c93-9b09-0139737a4e46.jpg)

**Comment by allesclip** [comment]
Hallo,
hast du aktuell ein funktionierendes Script? 
Ich habe einen solchen Zähler bekommen.

VG

**Comment by Marc0803** [comment]
Hallo,

das ist mein Script. Läuft seit dem völlig störungsfrei.

>D
>B
=>sensor53 r
>M 1
+1,3,s,0,9600,Haus
***@***.***,Aktueller Verbrauch,W,Power,0
***@***.***,Bezug,kWh,Bezug,2
***@***.***,Lieferung,kWh,Lieferung,2
#

Marc

Am Sa., 8. Juni 2024 um 19:25 Uhr schrieb allesclip <
***@***.***>:

**Comment by pille89** [comment]
Hallo Zusammen,

hat jemand auch schon geschaft das ganze per OBIS auszulesen? 
Das Script ist ja über SML.

Danke

Liebe Grüße

**Comment by Mattes1000** [comment]
@Marc0803: Da oben die Formatierung von Markdown angepasst wurde: 
So habe ich mein Skript angepasst
PS: Mit einem dreifachen ` vor und danach kann ein Codeblock erzeugt werden

```
>D
>B
=>sensor53 r
>M 1
+1,3,s,0,9600,Haus
1,77070100100700ff@1,Momentan Leistung,W,Leistung,0
1,77070100010800ff@1000,Totaler Verbrauch,KWh,Total_in,6
1,77070100020800ff@1000,Totale Erzeugung,KWh,Total_out,6
#
```
